### DISCUSSION TITLE
在V1中，SchemaMarkupField怎么添加onClick事件到ant-form-item上？

### DISCUSSION THREAD

**Discussion by ChimingPhang** [alibaba/formily#1975] [Q&A 1.x] [answered]
如题，在V1中，SchemaMarkupField怎么添加onClick事件到ant-form-item上？
![image](https://user-images.githubusercontent.com/3283625/128713183-0c2dc273-3462-48ff-a76f-17eb69da8251.png)
这样只能加到input 表单组件里面，希望点击整块添加active样式
![image](https://user-images.githubusercontent.com/3283625/128713287-83ae5d2d-7ab6-40bf-a118-d29f21928977.png)

**Comment by janryWang** [comment, marked as answer]
1.x设计上的问题，目前是不支持的

**Reply by ChimingPhang** [reply]
好的

**Reply by ChimingPhang** [reply]
有其他方法曲线实现吗？大佬
